test/hotspot/jtreg/applications/jcstress/JcstressRunner.java

```diff
@@ -104,11 +108,26 @@ private static String[] getCmd(String[] args) {
 
         extraFlags.add("--jvmArgs");
         extraFlags.add("-Djava.io.tmpdir=" + System.getProperty("user.dir"));
+
+        // The "default" preset might take days for some tests
+        // so use sanity testing by default.
+        String mode = "sanity";
         for (String jvmArg : Utils.getTestJavaOpts()) {
+            if(jvmArg.startsWith("-D" + MODE_PROPERTY)) {
+                String[] pair = jvmArg.split("=", 2);
+                mode = pair[1];
+                continue;
+            }
             extraFlags.add("--jvmArgs");
             extraFlags.add(jvmArg);
         }
 
+        extraFlags.add("-m");
+        extraFlags.add(mode);
+
+        extraFlags.add("-sc");
+        extraFlags.add("false");
```

Comment:
Is this to shorten the execution time? I'd recommend `-af GLOBAL` too then, since we are reducing the test matrix for it anyway.

Reply:
fixed